chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
I wondered if all serializers can save `None` as is.
I thought it is safer to save integer instead.
If all seralizers support saving `None`, `previous_epoch_detail` should be initialized with `None`.